### PR TITLE
Display list

### PR DESCRIPTION
**Pull request by EllieHield**
Added BookList and Book components within App.
Setup Ducks style Redux.
Added test setup with Enzyme.
Wrote initial unit tests.

### CODE REVIEW

**Comment by EllieHield** [comment]
@KMackow  @d-withers we deleted all the unnecessary files and re-committed

File: src/main/client/src/__tests__/bookList/BookList.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    publishDate: "1997"
+}
+const testBook2 = {
+    title: "Lord of the Rings",
+    id: 1,
+    author: "JRR Tolkien",
+    isbn: "1000000001",
+    publishDate: "1954"
+}
+
```
Comment (d-withers) [inline src/main/client/src/__tests__/bookList/BookList.test.js]: Should there also be a test for the empty list?
